docs/csaf_provider.md

```diff
@@ -106,6 +106,17 @@ The following example file documents all available configuration options:
 # If a list item starts with `expr:`
 #   the rest of the string is used as a JsonPath expression
 #   to extract a string from the incoming advisories.
+#   If the result of the expression is a string this string
+#   is used. If the result is an array each element of
+#   this array is tested if it is a string or an array.
+#   If this test fails the expression fails. If the
+#   test succeeds the rules are applied recursively to
+#   collect all strings in the result.
+#   Suggested expressions are:
+#   - vendor, product family and product names:  "expr:$.product_tree..branches[?(@.category==\"vendor\" || @.category==\"product_family\" || @.category==\"product_name\")].name"
+#   - CVEs: "expr:$.vulnerabilities[*].cve"
+#   - CWEs: "expr:$.vulnerabilities[*].cwe.id"
+#  s implementation of JsonPath expressions does not support the use of single-quotes. Double quotes have to be quoted. 
```

Comment:
```suggestion
#   This implementation of JsonPath expressions does not support 
#   the use of single-quotes. Double quotes have to be quoted.

Reply:
The `This` was only an `s`. And I guess we need to make a line cahgne to make golint happy ;-)